### ISSUE TITLE
Pop audio cut whilst moving bars and playing.

### ISSUE THREAD

**Issue by thoth-amon** [rncbc/padthv1#12]
Caused whilst using the bars to change the waveform and playing at the same time. When the mouse hold is lifted,  there is a loud pop whilst the audio cuts.  I have attached a recording of it. 

[padpop.zip](https://github.com/user-attachments/files/16033018/padpop.zip)

System Details: padthv1 verson 1.0  OpenSuse Tumbleweed (geekosdaw) Tested with the Standalone and LV2 in Carla and had the same issue.

**Comment by rncbc** [comment]
well yes, that's somehow expected behavior: editing the harmonics will regenerate a new brand wavetable, so probably all currently sounding voices, if any, will get abruptly cut off.

**Comment by thoth-amon** [comment]
The audio cutting out alone wouldn't be a problem if it didn't make the type of loud crack, which at high enough volume, can cause the audio server to die and/or possibly kill speakers.

**Comment by rncbc** [comment]
ok. please test whether 355a327 (develop branch) is any better.

thanks

**Comment by thoth-amon** [comment]
It is better but still a problem.  It's almost completely better when holding a note and then moving the bar/setting but if tapping on notes and moving settings it now varies from slightly to just as bad as before.  

[padclip.zip](https://github.com/user-attachments/files/16042208/padclip.zip)

**Comment by rncbc** [comment]
thanks; let me know whether ec06127 is an improvement...

unfortunately it won't get much better than this any time soon--changing most of the GEN PADsynth wavetable parameters are not realtime-smoothed anyway and probably never will; so sorry

**Comment by thoth-amon** [comment]
Still the same as before.

**Comment by rncbc** [comment]
last chance with 697ad93

nb. previous commits have been squashed; please reset to origin/develop.

**Comment by thoth-amon** [comment]
Not noticeably different again.  Still much better than originally though.

I don't know whether this is of any use but I did notice now that when the high pass filter is set to 30, or higher, that removes any remaining popping sounds when changes are made.

**Comment by rncbc** [comment]
addressed and prolly solved by now >= v1.0.0.5git.[1c5e17](https://github.com/rncbc/padthv1/commit/1c5e17d)

please test && tell

**Comment by thoth-amon** [comment]
Yes, the pop effect has now gone.  As you now have to wait for the previous note to finish, for sounds with long release this could feel like a change hasn't happened if you didn't know to wait. That's far less of a problem than the popping though IMHO
